frontend/src/views/EmulatorJS/Player.vue

```diff
@@ -88,6 +89,7 @@ declare global {
 }
 
 const supportedCores = getSupportedEJSCores(romRef.value.platform_slug);
+window.EJS_DEBUG_XX = true;
```

Comment:
should this always be on, even for regular use?

Reply:
I'm not OP, but reading the [documentation](https://emulatorjs.org/docs/options#ejs_debug_xx:~:text=Available%20options%20and,will%20be%20supported.), it doesn't seem it to me. It may be that it was left in for further testing?

Reply:
Definitely not, removed